project/plugin-scalajs.sbt

```diff
@@ -1,2 +1,2 @@
-addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "1.0.1")
+addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "1.3.0")
```

Comment:
Scala.js tests were failing for me in Scala 2 without this upgrade.  It's not germane to this ticket.